src/main/kotlin/org/opensearch/commons/alerting/action/DeleteCommentResponse.kt

```diff
@@ -0,0 +1,32 @@
+package org.opensearch.commons.alerting.action
+
+import org.opensearch.commons.alerting.util.IndexUtils
+import org.opensearch.commons.notifications.action.BaseResponse
+import org.opensearch.core.common.io.stream.StreamInput
+import org.opensearch.core.common.io.stream.StreamOutput
+import org.opensearch.core.xcontent.ToXContent
+import org.opensearch.core.xcontent.XContentBuilder
+
+class DeleteCommentResponse : BaseResponse {
```

Comment:
Does this need to extend `ActionResponse` instead of `BaseResponse`?

Reply:
I modeled after `DeleteMonitorResponse`, which extends `BaseResponse`. We can discuss offline the differences and which one's better.